### ISSUE TITLE
Too many versions

### ISSUE THREAD

**Issue by Axarva** [sarweshparajuli/nativefier-appimage#1]
Too many versions. Poor me can not decide what to install :(

**Comment by sarwesh00** [comment]
Make sure, you have your distro specific prerequisites installed.

And then run `npm install -g appnativefy` as superuser, to install the latest version.

The latest version is recommended, as it contains bug fixes.

[![npm version](https://badge.fury.io/js/appnativefy.png)](https://badge.fury.io/js/appnativefy)

**Comment by Axarva** [comment]
What are my distro-specific prerequisites, my distro isn't there on the install section :(

**Comment by sarwesh00** [comment]
Refer to this article [https://nodejs.org/en/download/package-manager/](https://nodejs.org/en/download/package-manager/) to install via your package manager.

For, wget it is highly likely it is already installed, else check it out in your package manager.

[![npm version](https://badge.fury.io/js/appnativefy.png)](https://badge.fury.io/js/appnativefy)

**Comment by Axarva** [comment]
Why do you keep adding in  [![npm version](https://camo.githubusercontent.com/feab53a73e85d6c230514a54235299776e13edab7bbb9a4d3dfb36c6cecbb8b6/68747470733a2f2f62616467652e667572792e696f2f6a732f6170706e617469766566792e706e67)](https://badge.fury.io/js/appnativefy) :(

**Comment by sarwesh00** [comment]
Well, that is just for ease, if there aren't any more issues, i'll proceed to close this?

**Comment by Axarva** [comment]
I see, please proceed. I shall proceed on my endeavors to learn the mysteries of nature: precisely why [![npm version](https://camo.githubusercontent.com/feab53a73e85d6c230514a54235299776e13edab7bbb9a4d3dfb36c6cecbb8b6/68747470733a2f2f62616467652e667572792e696f2f6a732f6170706e617469766566792e706e67)](https://badge.fury.io/js/appnativefy) keeps appearing on each of your comments. ~~I shall open up a new issue, and continue to learn.~~ Wait, no, Imma do this here.

TL;DR RIP bruhhhh

**Comment by Axarva** [comment]
*How is it just for ease?* :eyes:

**Comment by sarwesh00** [comment]
If you click on it, you reach the NPM page.
